front/src/applications/editor/Map.tsx

```diff
@@ -143,9 +147,11 @@ const MapUnplugged: FC<MapProps> = ({
 
         {/* Common layers */}
         <Background colors={colors[mapStyle]} />
+        <OrthoPhoto layerOrder={LAYER_GROUPS_ORDER[LAYERS.PLATFORMS.GROUP]} />
         <OSM mapStyle={mapStyle} />
         <Hillshade mapStyle={mapStyle} />
-        <Platform colors={colors[mapStyle]} />
+        <VirtualLayers />
```

Comment:
It has to be the very first layer you had inside the `Map` children

Reply:
For the moment I'll keep this order because all the other layers in editor's map are not ordered (and not switchable), it's a compromise until OuestWare devs add orderedlayers in their code. Thanx !